index.html

```diff
@@ -67,9 +66,9 @@
             <span class="icon icon-slider icon-slider_prew"></span>
             <span class="icon icon-slider icon-slider_next"></span>
         </div>
-        <div class="time">
-            <div class="time__clock"></div>
-            <div class="time__date"></div>
+        <div class="time__wrapper">
+            <div class="time"></div>
```

Comment:
do you use BEM? if yes, then it looks like incorrect block/element nesting. it is supposed to be 
```
.block
.block__element
```